projects/ekg.json

```diff
@@ -1,8 +1,11 @@
 {
   "name_cs": "Predikce srdečních arytmií",
   "name_en": "Heart arrhythmia prediction",
+  "name_de": "Vorhersage von Herzrhythmusstörungen",
```

Comment:
Vorhersage je spis vestba, ne? Mozna Prognose nebo Prediktion nebo tak neco?

Reply:
třeba Wetter Vorhersage je běžně používaný termín, takže asi v pohodě. I když je to taky trochu věštba :-)

Reply:
Díky za opravu. Napadlo mě `Vorhersage`, protože je to slovo právě díky předpovědi počasí často používané. Pro oblast medicíny je `Prognose` určitě vhodnější.